app/views/claims/schools/claims/_mentors_form.html.erb

```diff
@@ -0,0 +1,41 @@
+<% content_for :page_title, t(".title") %>
+
+<%= content_for(:before_content) do %>
+  <%= govuk_back_link(href: claims_school_claim_path(id: claim.id)) %>
+<% end %>
+
+<%= form_with(
+  model: claim,
+  url: update_claims_school_claim_path(claim.school_id, claim, "mentors"),
+  data: { turbo: false },
+) do |f| %>
+
+  <%= f.govuk_error_summary %>
+
+  <h1 class="govuk-heading-l"><%= t(".heading") %></h1>
+
+  <%= f.fields_for :mentor_trainings do |mentor_form| %>
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <div class="govuk-form-group">
+
+          <%= mentor_form.govuk_collection_select(
+            :teacher_id,
+            Teacher.all,
```

Comment:
I think Mentor should additionally belong to a School. (As per the [data model document](https://github.com/DFE-Digital/itt-mentor-services/blob/beaa69d2273f9db16b52e341ed04d519675b3517/docs/data-model.md).)

That way we'd be able to scope this list of Mentors to only those who belong to the current School.

```suggestion
            @school.mentors,
```